samples/compare/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'kotlin-kapt'
```

Comment:
org.jetbrains.kotlin.kapt

Also apply .jvm first?

Reply:
Wait you're not even using kapt. Just jvm then?

Reply:
Copy pasta error. Removed!